lib/getWorkflow.js

```diff
@@ -9,80 +9,203 @@
 const filterNodeName = name =>
     (/^~(pr|commit|release|tag|sd@)/.test(name) ? name : name.replace('~', ''));
```

Comment:
```suggestion
    (/^~((pr|commit|release|tag)(?:$|:)|(sd@))/.test(name) ? name : name.replace('~', ''));
```

Might fix https://github.com/screwdriver-cd/screwdriver/issues/1417

Reply:
Can fix in a separate PR

Reply:
opened a PR: https://github.com/screwdriver-cd/workflow-parser/pull/28